teamworkApp/lib/evaluateAnswers.py

```diff
@@ -0,0 +1,49 @@
+from enum import IntEnum
+import sqlite3
+
+DB = 'db/development.sqlite3'
+
+Style = IntEnum('Style', 'Contributor, Collaborator, Communicator, Challenger', start=0)
+Answer_Value = IntEnum(
+    'Answer_Value', 
+    '1234 1243 1324 1342 1423 1432 2134 2143 2314 2341 2413 2431 3124 3142 3214 3241 3412 3421 4123 4132 4213 4231 4312 4321',
+    start=0
+)
```

Comment:
I think that we repeat this in several files. I can pull out lines 4 to 11 and put them in a separate file which then gets imported where necessary.

Reply:
Definitely agree!